js/index.js

```diff
@@ -37,6 +37,73 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+// helper functions 
+
+function ObjToArray(array){
+  return Object.entries(array); // returns array of key pair values
+}
+
+function isContentHeader(value){
+  return value[0].includes("h4");
+}
+
+function notContentHeader(value) {
+  return !value[0].includes("h4")
+}
+
+function populateHeaderMenu(menu, image) {
+
+}
+
+function populateMainContent(oldContent, newHeaders, newText) {
+  let headerContent = oldContent.querySelectorAll(".text-content h4")
+  for (let i = 0; i <= headerContent.length; i++) {
+    headerContent[i].textContent = newHeaders[i];
+  }
+
+  let textContent = oldContent.querySelectorAll(".text-content p")
+  for (let i = 0; i <= headerContent.length; i++) {
+    headerContent[i].textContent = newText[i];
+  }
+}
+
+// Step 1 // declare data & relevant properties 
+
+let navItems = siteContent.nav;
+
+let header = document.querySelector("header");
```

Comment:
I want you to think about how you might implement a more concise approach, especially in this main content part, but wow - this is incredible that you were able to do this in the time and just have this approach and logic is mindblowing - job well done.

Reply:
thank you, i'm going to explore being more concise!